ui/ui-components/components/property/profilePreview.tsx

```diff
@@ -178,7 +185,7 @@ export default function ProfilePreview(props) {
             <ListGroup.Item key={`profile-preview-row-${k}`} variant="light">
               <strong>{k}</strong>:{" "}
               {otherProperties[k]?.type === "date"
-                ? new Date(otherProperties[k]?.values[0]).toLocaleString()
+                ? calculatePreviewDate(otherProperties[k]?.values[0])
```

Comment:
I'm not sure if this incoming value can be typed.

Reply:
`otherProperties[k]?.values[0]` should always be a number here if you want to check for that